Apps/WebClient/test/functional/e2e/cypress/integration/healthinsights/insights.js

```diff
@@ -1,15 +1,17 @@
 const { AuthMethod } = require("../../support/constants")
 
 describe('Health Insights', () => {
-    beforeEach(() => {
+    before(() => {
+        cy.server()
+        cy.fixture('insightsConfig').as('config')
+        cy.route('GET', '/v1/api/configuration/', '@config')
         cy.login(Cypress.env('keycloak.username'), 
                  Cypress.env('keycloak.password'), 
                  AuthMethod.KeyCloak)
+        cy.get('#menuBtnHealthInsights').click()
     })
 
-    it('Test 1', () => {
-    })
-
-    it('Test 2', () => {
+    it('totally has 259 records', () => {
```

Comment:
The it should document the test, that is the what.  This doesn't tell us anything.

This applies to many of your tests.

Reply:
@sslaws I have updated the test to "Total records count (for medication) should not have 0 records.", can you please review it again?